koku/common/utils.py

```diff
@@ -0,0 +1,97 @@
+import functools
```

Comment:
The name of this file could use some improvement. `utils` doesn't tell you anything about what might be in the file.

Also, given that this retry function was written for Trino, and explicitly has `HIVE` in the function, this probably ought to be moved to the `trino_database.py` file.